packages/lib-utils/src/k8s/hooks/useK8sWatchResource.ts

```diff
@@ -32,17 +31,13 @@ const NOT_A_VALUE = '__not-a-value__';
 export const useK8sWatchResource = <R extends K8sResourceCommon | K8sResourceCommon[]>(
   initResource: WatchK8sResource | null,
 ): WatchK8sResult<R> => {
-  const cluster = useSelector<SDKStoreState, string>((state) => getActiveCluster(state));
```

Comment:
Discussed with Vidya offline - stripping out the `activeCluster` logic is going to create a gap with the core SDK and the console SDK. For now we are ok with this as there is no immediate requirement from hac-core/hac-dev for this.

@vidyanambiar did we create a story under https://issues.redhat.com/browse/HAC-557?

**EDIT** -- looks like we have one already here: https://issues.redhat.com/browse/HAC-1168 for "Implementing usage of active cluster and user"

Reply:
Yes @florkbr - I had created that one: https://issues.redhat.com/browse/HAC-1168